src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -190,42 +203,36 @@ void HRVOAgent::computeNewVelocity()
     candidates_.insert(std::make_pair(
         (pref_velocity_ - candidate.position_).lengthSquared(), candidate));
 
+    // this adds candidate points that are projections of the preferred velocity onto each
+    // line segment
```

Comment:
We should clarify that the line segment is the Velocity Obstacle side/edge

Reply:
updated